client/src/hooks.js

```diff
@@ -52,4 +54,23 @@ export const CheckIfUserCanAccessPage = () => {
 			history.push(HOME, 'openRegister');
 		}
 	}, [isAuthenticated, history]);
-};
\ No newline at end of file
+};
+
+export const useInitGA = () => {
+	useEffect(() => {
+		if (process.env.CONFIG.analytics.ga.enabled) {
+			ReactGA.initialize(process.env.CONFIG.analytics.ga.ua);
+		}
+	}, []);
+};
+
+export const useInitHotjar = () => {
+	useEffect(() => {
+		if (process.env.CONFIG.analytics.hotjar.enabled) {
+			hotjar.initialize(
+				process.env.CONFIG.analytics.hotjar.hjid,
```

Comment:
please consider using clearer names for config, require them before initizializing . Also, maybe we could consider a 'config' service, that would handle getting config values from `process.env` or other resources, so it can handle it with default values, to prevent exceptions (if config not exist etc)

Reply:
the default.json values are the default ones and the local.json are overrides. this is the same mechanism that we are using in our server.